internal/ais/workflow.go

```diff
@@ -188,6 +188,20 @@ func (w *Workflow) SessionHandler(ctx temporalsdk_workflow.Context, aipUUID, aip
 		return "", e
 	}
 
+	var combineMDResult CombineMDActivityResult
+	e = temporalsdk_workflow.ExecuteActivity(
+		withFilesystemActivityOpts(ctx),
+		CombineMDActivityName,
+		&CombineMDActivityParams{
+			AreldaRelPath: metadataRelPath,
+			METSRelPath:   metsPath,
+			WorkingDir:    filepath.Join(localDir, filepath.Base(metadataRelPath)),
```

Comment:
Just use `localDir`, I think we don't need an extra folder with the name of the metadata file.